tensornetwork/backends/jax/jax_backend.py

```diff
@@ -49,4 +48,28 @@ def convert_to_tensor(self, tensor: Tensor) -> Tensor:
   def concat(self, values: Tensor, axis: int) -> Tensor:
     return numpy.concatenate(values, axis)
 
-  #TODO: add initializers (randn, zeros, eye, ones)
+  def randn(self,
+            shape: Tuple[int, ...],
+            dtype: Optional[numpy.dtype] = None,
+            seed: Optional[int] = None) -> Tensor:
+    if not seed:
+      seed = numpy.random.randint(0, 2**63)
+    key = self.jax.random.PRNGKey(seed)
+    if not dtype:
+      dtype = self.dtype
+    if not dtype:
+      dtype = numpy.float64
+    if dtype is self.np.complex128:
```

Comment:
Why did you duplicate this code for these two cases?

Reply:
differentn dtypes in jax.random.normal

Reply:
There is no need to duplicate the code. Refactor it by having the different types be variables. (Duplicating code is almost always bad practice since it can cause a lot of issues when you need to change/fix something later and you miss the duplication.)